docs/HardWonWisdom.md

```diff
@@ -99,3 +99,16 @@ A couple more observations:
 
 So, either by mimicking the approach of the existing `ValidityInterval` validation logic or by altering the STS innards to allow short-circuiting failure, we could reify the `EpochInfo` failures into the `PredicateFailure` hierarchy, and thereby leverage the types to force each invocation of the Ledger API to independently handle the possibility of `PastHorizonException`s.
 But it's not obvious that that is definitely worth the extra complexity it would introduce on the Ledger code.
+
+## Why use the Honest Chain Growth window as the Ledger's Stability Window?
+
+Suppose we have selected a different chain than our peer and that our selected chain has L blocks after the intersection and their selected chain has R blocks after the intersection.
+
+REQ1: If k<L, we must promptly disconnect from the peer (b/c of Common Prefix violation and Limit on Rollback).
```

Comment:
I wonder if we could add some ascii art to illustrate this, eg:

```text

|----------- R ------------|

          |------- k ------|

n ------- I ----------------
 \
  \-------------------------

|------------ R -----------|

where

- n: intersection point between our chain and our peer's.
- I: immutable point
```

Reply:
> because the intersection point lies before the immutable tip.

That fact only requires a disconnection because it represents a Common Prefix violation and/or usefully communicating with this peer would require violating our Limit on Rollback.

So I wouldn't want to replace the text with your suggestion, but I wouldn't object to adding it (and incorporating the sentence I just wrote above this one).

Thoughts?

Reply:
I hesitate to include a diagram in `HardWonWisdom`; seems out of scope/bloaty. But if this were integrated into a document with a wider intended audience, then I agree a diagram would make sense (but probably in a section explaining how we rely on Common Prefix to justify disconnections).

Reply:
> Thoughts?

Alright :) I was just pointing out that the situation you described is one in which the intersection point lies before the immutable tip.